docs/man/man1/kubectl-create-imported-service.1

```diff
@@ -0,0 +1,200 @@
+.TH "KUBERNETES" "1" " kubernetes User Manuals" "Eric Paris" "Jan 2015"  ""
+
+
+.SH NAME
+.PP
+kubectl create imported\-service \- Import an IP address external to Kubernetes and expose it as a new Kubernetes Service
+
+
+.SH SYNOPSIS
+.PP
+\fBkubectl create imported\-service\fP [OPTIONS]
+
+
+.SH DESCRIPTION
+.PP
+Take an existing external service (IP address for now) and import it as a Kubernetes service.
+
+.PP
+Creates a 'headless' Service with the given name, and creates an Endpoints object containing one or more IP addresses.
```

Comment:
"headless" isn't the correct terminology.

An "imported" service is one with a nil selector and manually populated Endpoints.

A "headless" service is one with no virtual clusterIP.

These are orthogonal features.